src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,27 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const bullsAndCows = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  const userNumbers = Array.from(String(userInput), (num) => +num);
+  const gameNumbers = Array.from(String(numberToGuess), (num) => +num);
+
+  for (let i = 0; i < 4; i++) {
+    const number = gameNumbers[i];
+
+    if (userNumbers.includes(number)) {
+      if (userNumbers.indexOf(number) === i) {
+        bullsAndCows.bulls += 1;
+      } else {
+        bullsAndCows.cows += 1;
+      }
+    }
```

Comment:
The current logic may incorrectly count a digit as a cow even if it has already been counted as a bull. Consider using separate loops or a mechanism to track which digits have been counted as bulls to prevent them from being counted as cows.